frontend/src/stores/MainStore.js

```diff
@@ -21,6 +21,8 @@ class MainStore {
     @observable hasEditSettings = false;
     @observable executionOutputs = null;
     @observable isUpdateComplete = false;
+    @observable showModal = false;
```

Comment:
showModal is not a good variable name; let's make it more precise `showWordReportOptionModal`